frontend/src/models/models.html

```diff
@@ -28,7 +28,7 @@ <h1>Models</h1>
           <th v-for="path in schemaPaths">
             {{path.path}}
             <span class="path-type">
-              ({{(path.instance || 'unknown').toLowerCase()}})
+              ({{(path.instance ? path.instance : isSubdoc(path) ? 'subdocument' : 'unknown').toLowerCase()}})
```

Comment:
Shouldn't put this much logic into a template. Ideally you would move all this into a method. But I'll suggest an alternative fix for this particular issue.